### DISCUSSION TITLE
Categories in hugo blog awesome

### DISCUSSION THREAD

**Discussion by Nrosa01** [hugo-sid/hugo-blog-awesome#151] [Q&A]
Hi! I've just discovered hugo and this theme and I'm enjoying both the framework and this theme (sorry for redundancy). But I'd like to know if this theme support categories. In one of the examples, I saw a tag metadata in the frontmatter but that doesn't seem to affect the page. Is it something for SEO?

If this is not a current feature please tell me so then I'll make a request in [feature requests](https://github.com/hugo-sid/hugo-blog-awesome/discussions/categories/feature-requests)

**Comment by hugo-sid** [comment]
> I'm enjoying both the framework and this theme

That's great @Nrosa01.

> a tag metadata in the frontmatter but that doesn't seem to affect the page

Thanks for bringing this up. I will look into it.

> Is it something for SEO

No, it is supposed to functions like a [Hugo taxonomy](https://gohugo.io/content-management/taxonomies/).

> If this is not a current feature please tell me so then I'll make a request

You're right. This is not a current feature.
